ArcadiaTeamsBot/Controllers/BotController.cs

```diff
@@ -11,8 +11,8 @@
     [Route("api/messages")]
     public class BotController : ControllerBase
     {
-        private readonly IBotFrameworkHttpAdapter botAdapter;
         private readonly IBot bot;
+        private readonly IBotFrameworkHttpAdapter botAdapter;

```

Comment:
Please, switch off ordering of members alphabetically, it is annoying.

Reply:
Turned off. It will no longer bother

Reply:
Switched off, but not committed. I suppose it should be saved in editor config.